src/debug/ee/arm64/primitives.cpp

```diff
@@ -52,4 +52,5 @@ bool IsSSFlagEnabled(DT_CONTEXT *pContext, Thread *)
 {
     return IsSSFlagEnabled(pContext);
 }
+#error zdfglhjksd
```

Comment:
Is this intended?

Reply:
It was stray debugging code that accidentally got checked in._